orbs/shared/jobs/test.yaml

```diff
@@ -0,0 +1,102 @@
+description: Run tests on the current commit
+parameters:
+  app_name:
+    description: Name of the application
+    type: string
+  docker_username:
+    default: _json_key
+    description: Username to use when fetching images from a registry
+    type: string
+  docker_password:
+    default: $GCLOUD_SERVICE_ACCOUNT
+    description: Environment variable to read docker password from
+    type: string
+  postgres_version:
+    default: "13.3"
+    description: Postgres version to run
+    type: string
+  mysql_version:
+    default: "8.0"
+    description: MySQL version to run
+    type: string
+  redis_version:
+    default: "6.0"
+    description: Redis version to run
+    type: string
+  kafka_version:
+    default: "2.3"
+    description: Kafka version to run
+    type: string
+  minio_version:
+    default: bootstrap
+    description: Minio version to run
+    type: string
+  localstack_version:
+    default: "0.12.11"
+    description: Localstack version to run
+    type: string
+  localstack_services:
+    default: dynamodb
+    description: Services to run on localstack, comma separated
+    type: string
+
+# DEPRECATED: This will be removed in a future release. e2e should be used
+# for integration+e2e tests, NOT in CircleCI.
+docker:
```

Comment:
we always have to run all of these? couldn't we just run them conditionally for now

Reply:
CircleCI doesn't let you conditionally run docker containers :(

Reply:
dumb but ok

Reply:
This will be a breaking change to use in services so this likely won't ever be used in it's current form outside of our projects that will use this

Reply:
until the next window, that is